### PR TITLE
#161348763 Article tags should be present in all relevant views

### PR DESCRIPTION
**Pull request by RachaelNantale**
#### What does this PR do?
Users can view the various article tags on an article.
A user can click on view all the tags available.

#### Description of Task to be completed?
- Users can view the various article tags on an article.
- A user can click on view all the tags available.

#### How should this be manually tested?
- Go to the article's URL: `/articles/:articleId` e.g. `/articles/40` and `/articles`
- You should be able to see the  article tags  on the article
- You should see the popular articles.

#### Any background context you want to provide?
You do need to be logged in to add a tag to an article
#### What are the relevant pivotal tracker stories?
[#161348763](https://www.pivotaltracker.com/story/show/161348763)

#### Relevant Screenshots
screen shot 2018-12-18 at 9 53 02 am

### CODE REVIEW

**Review by brucemakallan — DISMISSED** [review]
Please add the most recent screenshots, showing tags on the homepage.

**Review by brucemakallan — DISMISSED** [review]
Thanks for working on the changes I suggested.

**Review by dnuwa — DISMISSED** [review]

**Review by brucemakallan — DISMISSED** [review]

**Review by dnuwa — DISMISSED** [review]

**Review by brucemakallan — DISMISSED** [review]

**Review by dnuwa — DISMISSED** [review]

**Review by brucemakallan — DISMISSED** [review]

**Review by judeinno — DISMISSED** [review]

**Review by judeinno — DISMISSED** [review]

**Comment by dnuwa** [comment]
@RachaelNantale, please work on the conflicts

**Review by judeinno — DISMISSED** [review]

**Review by dnuwa — DISMISSED** [review]

File: src/actions/tagsActions.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+});
+
+export const fetchTagFailure = tags => ({
+  type: ACTION_TYPE.FETCH_TAGS_FAILURE,
+  tags,
+});
+
+export const getTag = () => (dispatch) => {
+  const token = localStorage.getItem('token');
+  // axios.defaults.headers.common.Authorization = `Token ${token}`;
```
Comment (tibetegya) [inline src/actions/tagsActions.js]: remove commented out code

File: src/components/Home/index.js [hunk 2]
```diff
@@ -17,7 +18,11 @@ const Home = () => (
           <Route path="/" component={Articles} />
         </div>
         <div className="righter">
-          <div className="righter-title"><h3>Popular Tags</h3></div>
+          <div>
+            <div className="righter-title">Popular Tags </div>
+            {/* <Route path="/tags" component={Tags} /> */}
```
Comment (tibetegya) [inline src/components/Home/index.js]: remove commented out code

File: src/reducers/socialLoginReducer.js [hunk 3]
```diff
@@ -7,8 +7,8 @@ const initialState = {
 
 const socialLoginReducer = (state = initialState, action) => {
   switch (action.type) {
-    case ACTION_TYPES.SOCIAL_LOGIN:
-      return { ...state };
+    // case ACTION_TYPES.SOCIAL_LOGIN:
```
Comment (tibetegya) [inline src/reducers/socialLoginReducer.js]: remove this

**Review by judeinno — APPROVED** [review]

**Review by brucemakallan — APPROVED** [review]

**Pull request merged**